packages/editor/src/client/atoms/searchAndFilter/listViewFilters.tsx

```diff
@@ -0,0 +1,255 @@
+import CloseIcon from '@rsuite/icons/legacy/Close'
+import React, {useEffect, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Button, DateRangePicker, Form, Input, SelectPicker, Toggle} from 'rsuite'
+
+import {
+  ArticleFilter,
+  AuthorRefFragment,
+  DateFilterComparison,
+  PageFilter,
+  PeerWithProfileFragment,
+  usePeerListQuery
+} from '../../api'
+import {AuthorCheckPicker} from '../../panel/authorCheckPicker'
+
+const formInputStyle = {
+  marginRight: '15px',
+  marginTop: '0',
+  marginBottom: '10px'
+}
+
+type Field =
+  | 'title'
+  | 'preTitle'
+  | 'lead'
+  | 'description'
+  | 'draft'
+  | 'published'
+  | 'pending'
+  | 'authors'
+  | 'peer'
+  | 'publicationDate'
+
+export interface ListViewFiltersProps {
+  fields: Field[]
+  filter: Partial<ArticleFilter & PageFilter>
+  isLoading: boolean
+  onSetFilter(filter: ArticleFilter & PageFilter): void
+  setPeerFilter?(value: string): void
+}
+
+export function ListViewFilters({
+  fields,
+  filter,
+  onSetFilter,
+  setPeerFilter
+}: ListViewFiltersProps) {
+  const {t} = useTranslation()
+  const [resetFilterKey, setResetFilterkey] = useState<string>(new Date().getTime().toString())
+  const [allPeers, setAllPeers] = useState<PeerWithProfileFragment[]>([])
```

Comment:
Should be a useMemo or a simple const (const preferred).

```
const allPeers = useMemo(() => peerListData?.peers, [peerListData?.peers])
// or even better
const allPeers = peerListData?.peers;
```

As this double renders